src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/AbstractItemNetwork.java

```diff
@@ -477,4 +505,21 @@ private void filter(@Nullable ItemStack stack, List<ItemStackAndInteger> items,
         }
     }
 
+    @Nonnull
+    protected ItemFilter getItemFilter(@Nonnull Block node) {
+        Location loc = node.getLocation();
+        ItemFilter filter = filterCache.get(loc);
+
+        if (filter == null) {
+            ItemFilter newFilter = new ItemFilter(node);
+            filterCache.put(loc, newFilter);
+            return newFilter;
+        } else if (filter.isDirty()) {
+            filter.update(node);
+            return filter;
+        } else {
+            return filter;
```

Comment:
Not sure what the default here is
```suggestion
        } else if (filter.isDirty()) {
            filter.update(node);
        }
        return filter;
```

Reply:
Just no...

Reply:
lol